website/project/model.py

```diff
@@ -529,6 +524,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     #: Whether this is a pointer or not
     primary = True
 
+    is_draft_registration = False
```

Comment:
Changing it, maybe. But making sure I understand the different use cases (and how flags go together) is very much in scope for the review- I'll cheerfully agree to making that distinction.